node/node.go

```diff
@@ -376,13 +376,24 @@ func (n *Node) obtainJWTSecret(cliParam string) ([]byte, error) {
 // startup. It's not meant to be called at any time afterwards as it makes certain
 // assumptions about the state of the node.
 func (n *Node) startRPC() error {
-	if err := n.startInProc(); err != nil {
+	// Filter out personal api
+	apis := n.rpcAPIs
+	for i, api := range apis {
+		if api.Namespace == "personal" {
+			if !n.config.EnablePersonal {
+				apis = append(apis[:i], apis[i+1:]...)
+			} else {
+				log.Error("Deprecated personal namespace activated")
```

Comment:
```
ERROR[01-17|10:32:36.036] Enabling deprecated personal namespace 
```
I think ERROR is misusing the log semantics.
```suggestion
				log.Warn("Deprecated personal namespace activated")
```